### PR TITLE
Adds Cli for microsoft 365 2024 wrap up

### PR DESCRIPTION
**Pull request by Adam-it**
## Category

- [x] New article

### CODE REVIEW

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+You may also check out the [Getting started with CLI for Microsoft 365 recording](https://www.youtube.com/watch?v=XFG8gVGvXpA).
+{{< youtube XFG8gVGvXpA >}}
+
+The third major release - v10 - was our regular planned major release in which we introduce breaking changes. This time we introduced 34 breaking changes that improved the overall experience of using CLI for Microsoft 365 and aligned it with latest naming in Microsoft 365 like renaming of AAD to Entra ID and also we finally switched from Yammer to Viva Engage
+If you are interested in the full list of changes and improvements, you can check out the [release notes](https://pnp.github.io/cli-microsoft365/v10-upgrade-guidance).
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:28]: let's either point to the release notes, or refer to it as upgrade guidance.

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+## Support for using multiple accounts
+
+Over the year we introduced many new features and improvements to the CLI for Microsoft 365 but the one that stand out most this year is the possibility to sign in to multiple accounts/tenants and then just switch between them without the need to sign out and sign in again. This feature is especially useful for admins who manage multiple tenants, those might by dev, test, and production environments or different client tenants. This is a significant quality-of-life improvement that makes everything much easier and faster.
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:32]:
```suggestion
Over the year we introduced many new features and improvements to the CLI for Microsoft 365 but the one that stand out the most this year is the possibility to sign in to multiple accounts/tenants and then just switch between them without the need to sign out and sign in again. This feature is especially useful for admins who manage multiple tenants, these might by dev, test, and production environments or different client tenants. This is a significant quality-of-life improvement that makes everything much easier and faster.
```

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+Check out [this recording](https://www.youtube.com/watch?v=vmtFRnNckGM) to see how it works.
+{{< youtube vmtFRnNckGM >}}
+
+To find out even more you may want to check the [m365 connection commands](https://pnp.github.io/cli-microsoft365/cmd/connection/connection-list)
+
+## Added new commands to manage SharePoint Premium and SharePoint Embedded
+
+This year we introduced many new commands but we are always looking for new areas to expand our CLI for Microsoft 365. All in all the aim is to have a single CLI to manage all Microsoft 365 services. This year we introduced new commands to manage [SharePoint Premium](https://adoption.microsoft.com/en-us/sharepoint-premium/start/) and [SharePoint Embedded](https://learn.microsoft.com/en-us/sharepoint/dev/embedded/overview).
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:41]:
```suggestion
This year we introduced many new commands but we are always looking for new areas to expand our CLI for Microsoft 365. All in all, the aim is to have a single CLI to manage all Microsoft 365 services. This year we introduced new commands to manage [SharePoint Premium](https://adoption.microsoft.com/en-us/sharepoint-premium/start/) and [SharePoint Embedded](https://learn.microsoft.com/en-us/sharepoint/dev/embedded/overview).
```

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+We are only getting started but the `m365 spe` commands will already allow you to manage basic operations on [container types](https://pnp.github.io/cli-microsoft365/cmd/spe/containertype/containertype-add) and [containers](https://pnp.github.io/cli-microsoft365/cmd/spe/container/container-get).
+
+Regarding SharePoint Premium we mainly introduced `m365 spp` commands that allow you to [document understanding models](https://pnp.github.io/cli-microsoft365/cmd/spp/model/model-get)
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:45]:
```suggestion
Regarding SharePoint Premium we mainly introduced `m365 spp` commands that allow you manage [document understanding models](https://pnp.github.io/cli-microsoft365/cmd/spp/model/model-get)
```

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+## Work done in numbers
+
+This year was very busy for us. To give you a better idea of how much work we have done, here are some numbers:
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:49]:
```suggestion
This year was very busy for us. To give you a better idea of the extent of our work, here are some numbers:
```

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+## Added new commands to manage SharePoint Premium and SharePoint Embedded
+
+This year we introduced many new commands but we are always looking for new areas to expand our CLI for Microsoft 365. All in all the aim is to have a single CLI to manage all Microsoft 365 services. This year we introduced new commands to manage [SharePoint Premium](https://adoption.microsoft.com/en-us/sharepoint-premium/start/) and [SharePoint Embedded](https://learn.microsoft.com/en-us/sharepoint/dev/embedded/overview).
+
+We are only getting started but the `m365 spe` commands will already allow you to manage basic operations on [container types](https://pnp.github.io/cli-microsoft365/cmd/spe/containertype/containertype-add) and [containers](https://pnp.github.io/cli-microsoft365/cmd/spe/container/container-get).
+
+Regarding SharePoint Premium we mainly introduced `m365 spp` commands that allow you to [document understanding models](https://pnp.github.io/cli-microsoft365/cmd/spp/model/model-get)
+
+## Work done in numbers
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:47]: @waldekmastykarz is it possible to share any usage stats?

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+- [Reshmee Auckloo](https://github.com/reshmee011)
+- [Saurabh Tripathi](https://github.com/Saurabh7019)
+- [Shantha Kumar T](https://github.com/ktskumar)
+- [Smita Nachan](https://github.com/SmitaNachan)
+- [Vedant Lakshminarayanan](https://github.com/Vedu1996)
+- [Waldek Mastykarz](https://github.com/waldekmastykarz)
+
+## 100th release!
+
+This year is special as we are celebrating the 100th release of CLI for Microsoft 365. Over the years, we’ve continually worked to enhance your experience, expand our capabilities, and keep pace with the evolving Microsoft 365 ecosystem.
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:87]:
```suggestion
This year is special as we are celebrating the 100th release of CLI for Microsoft 365. Over the years, enhanced your experience, expanded our capabilities, and kept pace with the evolving Microsoft 365 ecosystem.
```

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+Reaching this milestone is a testament to the amazing contributions of our community, the dedication of our team, and the ongoing support of our users. Thank you for being part of this journey—we’re excited to continue building a brighter future together!
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:89]:
```suggestion
Reaching this milestone is a testament to our community's contributions, our team's dedication, and our users' ongoing support. Thank you for being part of this journey—we’re excited to continue building a brighter future together!
```

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+To celebrate this milestone, we’ve prepared a special place in which we started gathering all the positive feedback and testimonials from our users. We’re thrilled to unveil our new community [testimonials page](https://pnp.github.io/cli-microsoft365/testimonials)! This page highlights the experiences and stories shared by users from around the world, showcasing the impact of CLI for Microsoft 365 in real-life scenarios. From solving challenges to streamlining workflows, these testimonials reflect the value the CLI brings to professionals and organizations alike.
+
+## Stay Connected!
+
+We value your feedback and are eager to hear from you. If you have any suggestions for improvement or want to engage with our community, you can reach out to us on [GitHub](https://github.com/pnp/cli-microsoft365/issues), [Discord](https://aka.ms/cli-m365/discord), or [Bluesky](https://bsky.app/profile/climicrosoft365.bsky.social)]. Don't hesitate to connect with us. Your input plays a vital role in shaping the future of CLI for Microsoft 365.
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:95]:
```suggestion
We value your feedback and are eager to hear from you. If you have any suggestions for improvement or want to engage with our community, you can reach out to us on [GitHub](https://github.com/pnp/cli-microsoft365/issues), [Discord](https://aka.ms/cli-m365/discord), or [Bluesky](https://bsky.app/profile/climicrosoft365.bsky.social). Don't hesitate to connect with us, as your input plays a vital role in shaping the future of CLI for Microsoft 365.
```

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+## Resources
+
+- [CLI for Microsoft 365 docs](https://pnp.github.io/cli-microsoft365/)
+- [CLI for Microsoft 365 GitHub repo](https://github.com/pnp/cli-microsoft365)
+- [CLI for Microsoft 365 npm package](https://www.npmjs.com/package/@pnp/cli-microsoft365?activeTab=readme)
+- [Microsoft 365 & Power Platform Community](https://pnp.github.io/#home)
+- [Join the Microsoft 365 & Power Platform Community Discord Server](https://discord.gg/YtYrav2VGW)
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:103]:
```suggestion
- [Join the Microsoft 365 & Power Platform Community Discord Server](https://aka.ms/cli-m365/discord)
```

**Review by milanholemans — CHANGES_REQUESTED** [review]
Well done @Adam-it! Made a few remarks while reading.

File: content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md (hunk 6 above)
Comment (Adam-it) [reply content/cli-for-microsoft-365/cli-for-microsoft-365-2024-wrap-up/index.md:47]: I added some basic info about our highest npm package download with link to stats from what we share on GH discussions

**Pull request merged**